sam/models/TimeseriesMLP.py

```diff
@@ -42,12 +42,12 @@ class TimeseriesMLP(BaseTimeseriesRegressor):
 
     Parameters
     ----------
-    predict_ahead: integer or list of integers, optional (default=1)
+    predict_ahead: integer or list of integers, optional (default=0)
```

Comment:
Given your other adjustments, shouldn't this be `[0]` by default? And is an integer even supported?

Reply:
Yes should be `[0]`. And integer is supported since it is changed to a list in the init.